notebooks/203-meter-reader/README.md

```diff
@@ -0,0 +1,17 @@
+# Industrial Meter Reader
+
+
+<p align="center" width="100%">
+    <img width="80%" src="https://user-images.githubusercontent.com/91237924/166135627-194405b0-6c25-4fd8-9ad1-83fb3a00a081.jpg"> 
+</p>
```

Comment:
Please use markdown instead of html tags

Reply:
thanks for you advise, I will update the notebook accordingly later